docs/swagger.yaml

```diff
@@ -522,50 +686,68 @@ definitions:
     - "status"
     properties:
       campaign:
-        $ref: "#/definitions/campaign"
+        $ref: "#/definitions/Campaign"
       currency:
         type: "string"
+        description: "ISO-4217: https://www.iso.org/iso-4217-currency-codes.html"
+        default: "NOK"
         enum:
         - "NOK"
+        minLength: 3
+        maxLength: 3
+        pattern: "^[A-Z[3]$"
         example: "NOK"
       id:
         type: "string"
         description: "Uniquely identifies this agreement"
+        maxLength: 36
         example: "agr_5kSeqzFAMkfBbc"
       interval:
         type: "string"
         description: "Interval for subscription"
+        default: "MONTH"
```

Comment:
Interval as a separate object in Swagger

Reply:
I understnd there are some special considerations for `DAILY`, because we need to inform the user about bupcoming charges, and there is an 8 day lead, etc. Should we wait with `DAILY` until we have an elegant way of providing it? Or at least document that `DAILY` is work in progress?